### DISCUSSION TITLE
Intermittent Fire Icon

### DISCUSSION THREAD

**Discussion by beewry** [AnySoftKeyboard/AnySoftKeyboard#2757] [Q&A] [answered]
This fire icon has been appearing and disappearing from the lower right hand corner of my keyboard. I cannot seem to find any explanation of what it means. Does anyone have any idea what it means?

![image](https://user-images.githubusercontent.com/40073590/110224020-55d23b00-7e95-11eb-8f0e-9df843064d28.png)

Thanks!

**Comment by menny** [comment, marked as answer]
It's a bug in our release process. It will be fix soon

**Reply by rivalcomp** [reply]
It's an issue again in the latest version 1.11.170 (LG V30, stock Android 9).

**Comment by timsterzel** [comment]
Since yesterday I have the same icon at the bottom right of my keyboard. Is it still just a bug or does it have any meaning?

**Comment by ivannunyadambiz** [comment]
so to sumarize, we have had our keyboard app invaded by

- the coronavirus
- a half moon
- fire
- stupid space punctuation "you typed this but i think you meant... instead" when using a shell

however there's no real response from the devs.

ok then.
might just switch to a different app.
